odoo/modules/loading.py

```diff
@@ -25,6 +28,49 @@
 _test_logger = logging.getLogger('odoo.tests')
 
 
+def _get_files_of_kind(kind, package):
```

Comment:
FWIW in a separate branch (which I never pushed because I never got it to a point of doing something *useful*) I moved these over to methods on the Node object, it really just shuffles the shit around but shortens ``loading.py` a tad and seems more logical as this literally only accesses the package & its parameter.

Reply:
Yeah it was your branch that tried to make module install/uninstall transactional right? I may copy over some of the code/commits *if* we end up going with this approach, rco suggested another approach that is maybe cleaner and doesn't require parsing the data files twice but I'm actually not sure if it'll work, to be seen.

Reply:
Yeah, I'm just suggesting taking advantage of you moving the functions around to moving them to Node directly, that seems like a good occasion / excuse to do that and move those bits outside of loading.